toptal/picasso
# style(Menu): set min width of dropdown menu item

[FX-3127]

### Description

Showcase how to use Dropdown menu items with descriptions. Also set `min-width` to comply with BASE. I asked Matt directly about this change and he agreed.

### How to test

- Check the [new example](https://picasso.toptal.net/fx-3127-dropdown-with-description/?path=/story/components-dropdown--dropdown#dropdown-menu) in the storybook and compare it with [the design](https://www.figma.com/file/5SCTOPrCDcHuk5We091GBn/Product-Library?node-id=202%3A10632). Be aware that the icon example will be done in the next PR.
- Check Happo screenshots

### Screenshots

![image](https://user-images.githubusercontent.com/6830426/197122697-5307480f-f531-4f0b-b535-2834a79c087b.png)

### Development checks



> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-3127]: https://toptal-core.atlassian.net/browse/FX-3127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
